extension-style/src/main/java/com/mapbox/maps/extension/style/expressions/generated/Expression.kt

```diff
@@ -129,7 +128,25 @@ class Expression : Value {
      *
      * @return Expression
      */
-    open fun build() = Expression(this)
+    open fun build(): Expression {
+      if (this.operator == "match") {
+        val newBuilder = ExpressionBuilder("match")
+        this.arguments.forEachIndexed { i, e ->
+          newBuilder.addArgument(
+            // https://github.com/mapbox/mapbox-maps-android/issues/965
+            // the match expression is an exception and it takes raw list instead of a list wrapped into
+            // literal expression when literal array is used as the label.
+            if (i % 2 == 1 && i != this.arguments.size - 1) {
```

Comment:
```suggestion
val lastIndex = this.arguments.size - 1
...
            if (i % 2 == 1 && i != lastIndex) {
```

maybe `arguments` list can be huge